### ISSUE TITLE
In Vocabulary, use "Object | Link" rather than "Link | Object" (Proposed Errata) 

### ISSUE THREAD

**Issue by bobwyman** [w3c/activitystreams#554]
Please Indicate One: 
 * [ x] Editorial
 * [ ] Question
 * [ ] Feedback
 * [ ] Blocking Issue
 * [ ] Non-Blocking Issue

Please Describe the Issue:

In the ActivityStreams Vocabulary's listing of Terms, the Range and Domain descriptions are defined sometimes as "Object | Link" and sometimes as "Link | Object." Given that "Object | Link" and "Link | Object" are semantically identical, unnecessary apparent specification complexity would be reduced by adopting a consistent method of expressing this conjunction.

I suggest that the Domain and Range descriptions of `attributedTo`, and `preview`; and the Range descriptions of `mediaType`, and `subject` should be changed from "Link | Object" to "Object | Link." Prior to these changes actually being made, they should be recorded as Errata.

Why "Object | Link" rather than "Link | Object"?
- "Object | Link" is used 25 times in the current version while "Link | Object" is used only 6 times. - The impact of this proposed change would be limited by adopting the form which is currently most commonly used.
- "Object" is more general than "Link" -  It seems natural to list options in decreasing order of generality, it makes sense to use "Object | Link" as the preferred pattern

**Comment by nightpool** [comment]
Since every errata has to be listed out and in theory read by all implementors I don't think this is worth the extra noise it would create in the errata list given that it makes no difference what order a union is in.

**Comment by evanp** [comment]
@bobwyman I agree with this as an editorial change, but I'm not sure it's an erratum. It's not actually an "error" -- just bad formatting! I think if we do an editorial update to the document, we should change this. I'm not sure how to track this, so I'm labelling it "editorial" and self-assigning. I'll see if I can find the right way to handle it.

**Comment by ThisIsMissEm** [comment]
Is this perhaps related to #572 ?
